mteb/tasks/Classification/yue/YueOpenriceClassification.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class YueOpenriceClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="YueOpenriceClassification",
+        description="A Cantonese dataset for sentiment classification on review",
+        reference="https://github.com/Christainx/Dataset_Cantonese_Openrice",
+        dataset={
+            "path": "izhx/yue-openrice-senti",
+            "revision": "b8b9a3d0e40bc0674945a0edb0bc549b554d6f20",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs={"yue": ["yue-Hant"]},
+        main_score="accuracy",
+        date=None,
```

Comment:
Time period for when the data was created was annotated as a tuple (from, to). Can be a guess.